jetty-ee11/jetty-ee11-servlet/src/test/java/org/eclipse/jetty/ee11/servlet/DefaultServletTest.java

```diff
@@ -124,7 +125,8 @@ public void init() throws Exception
 
         connector = new LocalConnector(server);
         connector.getConnectionFactory(HttpConfiguration.ConnectionFactory.class).getHttpConfiguration().setSendServerVersion(false);
-        Path extraJarResources = MavenPaths.findTestResourceFile(ODD_JAR);
+        Path extraJarResources = workDir.getPathFile("odd.jar");
+        Files.copy(Objects.requireNonNull(TestEeResources.getResourceAsPath(ODD_JAR)), extraJarResources);
```

Comment:
Perhaps the `TestEeResources.getResourceAsPath()` should throw an exception if the passed in reference isn't found, never returning null.

Just like `MavenPaths.findTest*` methods do.